common/v2/components/AccountList.tsx

```diff
@@ -275,21 +311,23 @@ const getSortingFunction = (sortKey: ISortTypes): TSortFunction => {
   }
 };
 
-function buildAccountTable(
+const buildAccountTable = (
   accounts: StoreAccount[],
   deleteAccount: (a: IAccount) => void,
   updateAccount: (u: TUuid, a: IAccount) => void,
   deletable?: boolean,
   favoritable?: boolean,
   copyable?: boolean,
+  privacyTriggerEnabled?: boolean,
```

Comment:
Perhaps we could call this something else? Calling it a trigger seems confusing?

Reply:
`privacyTriggerEnabled` => `privacyCheckboxEnabled` ?